src/main/pages/user-management/permissions.adoc

```diff
@@ -11,37 +11,35 @@ folder: user-management
 [id="overview"]
 == Overview
 
-Permissions are used to control user actions. Rather than providing a fixed set of roles we use a broader set of permissions that can be applied in any combination of objects to establish the security model you need.
+Permissions are used to control the actions of users and establish a security model. You can define _who_ is allowed to do _what_. You can control resources managed by Che and allow certain actions by assigning permissions to users.
```

Comment:
I feel that "You can define _who_ is allowed to do _what_" is redundant here.

Reply:
+1